alt-ergo-lib.opam

```diff
@@ -1,6 +1,6 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-version: "dev"
+version: "models"
```

Comment:
that should not be changed by this PR: the version should stay dev as this is going to be merged into `next` which is the dev version